doc/userguide/rules/http-keywords.rst

```diff
@@ -44,6 +44,7 @@ http.accept_enc                http_accept_enc (*)      Request
 http.referer                   http_referer (*)         Request
 http.connection                http_connection (*)      Both
 file.data                      file_data (*)            Both
+file.name                      filename                 Request
```

Comment:
It looks like `file.name` is a sticky buffer so should have the asterisk?

But I do find this confusing.  When I look at existing docs I see `file_data` as a legacy content modifier that *was* a sticky buffer, but the *new* keyword, `file.data` is not a sticky buffer.  At least that is how I read it, but I know that's not the case.

Reply:
The * usage is a bit confusing. I wonder since suri 4 is EOL and all the http keywords are sticky as far as I know. We could probably do away with those tables?

Reply:
Agreeing that the `*` is confusing.
Considering this is the new documentation, it should be fine...

I wonder, though, if this wouldn't be a bigger change than just this one we're thinking of now.

Reply:
Do you want a new ticket for the table change idea? I agree that is a bit too intrusive to include here probably.

Reply:
Probably better, indeed, as it would be good to also check if more such updates would be needed 🤔